csharp/src/Drivers/BigQuery/BigQueryConnection.cs

```diff
@@ -70,32 +70,30 @@ public BigQueryConnection(IReadOnlyDictionary<string, string> properties)
         /// <exception cref="ArgumentException"></exception>
         internal void Open()
         {
-            string projectId = string.Empty;
-            string clientId = string.Empty;
-            string clientSecret = string.Empty;
-            string refreshToken = string.Empty;
+            string? projectId = string.Empty;
+            string? clientId = string.Empty;
+            string? clientSecret = string.Empty;
+            string? refreshToken = string.Empty;
```

Comment:
Does it make more sense to initialize these to null now that they're marked as nullable?